README.md

```diff
@@ -182,8 +182,8 @@ A: Here are three:
       custom apps directory as necessary.
 
       This is documented in `NuttX/boards/README.txt` and
-      `nuttx/Documentation/NuttxPortingGuide.html` (Online at
-      https://bitbucket.org/nuttx/nuttx/src/master/Documentation/NuttxPortingGuide.html#apndxconfigs
```

Comment:
Just a note that we need to update all the documentation and bitbucket links in the app repo